packages/mdc-data-table/README.md

```diff
@@ -34,6 +34,19 @@ npm install @material/data-table
 > *NOTE*: Styles for any components you intend to include within data-table (e.g. Checkboxes, etc.) must also be
 > imported.
 
+### JavaScript Instantiation
+
+```js
+import {MDCDataTable} from '@material/data-table';
+const dataTable = new MDCDataTable(document.querySelector('.mdc-data-table'));
+```
+
+> See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
+
+> Instantiating `MDCDataTable` component is only required to add interactions for example, row selection.
+
+MDC Data Table component auto instantiates `MDCCheckbox` for header row checkbox and all row checkbox. Make sure to set required class names to instantiate checkbox component. We suggest to use `layout` API when rows are added or removed from data table to register new checkbox components.
```

Comment:
`... all row checkboxes...`

Reply:
all row checkboxes?

Reply:
` for header row checkbox and all row checkbox` --> ` for header row checkbox and all row checkboxes`